cmake/aarch64InstructionFlags.cmake

```diff
@@ -0,0 +1,121 @@
+include(CheckCXXCompilerFlag)
```

Comment:
As discussed, let's simplify and use only the latest arm machine for benchmarks at the moment